extensions/commands/art/README.md

```diff
@@ -84,3 +84,41 @@ conan art:build-info promote mybuildname_aggregated 1 origin-artifactory-repo de
 ```
 
 Now, both release and debug binaries from that pacakge ae available in the destination repository with just one command.
+
+#### [conan art:promote](cmd_promote.py)
+
+```
+$ conan art:promote -h
```

Comment:
Updating the help message is the only thing left 💪

Reply:
Good catch!